lab-kyler/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "15-basic_auth",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "DEBUG='kauth*' gulp",
```

Comment:
You are using your gulp file for npm run test but there are no gulp dependencies included in your package.json so npm run test throws a 'Cannot find module 'gulp-eslint' error. Your tests only pass when someone runs them with mocha.

You'll want to make sure you omit gulp in your test script if you don't want to use it anymore so npm run test works as expected with debug logs included 👍